CMakeLists.txt

```diff
@@ -2,14 +2,14 @@ cmake_minimum_required(VERSION 3.12)
 
 include(pico_sdk_import.cmake)
 
-set(FREERTOS_KERNEL_PATH ${CMAKE_CURRENT_LIST_DIR}/freertos)
+set(FREERTOS_KERNEL_PATH "${CMAKE_CURRENT_LIST_DIR}/freertos")
 include(FreeRTOS_Kernel_import.cmake)
 
 project(debugprobe)
 
 pico_sdk_init()
 
-if (${PICO_SDK_VERSION_MAJOR} LESS 2)
+if ("${PICO_SDK_VERSION_MAJOR}" LESS 2)
```

Comment:
Presumably there'll never be a space in `PICO_SDK_VERSION_MAJOR` ?

Reply:
Most likely. It was just quicker and easier to replace everything with quotations, and shouldn't (as far as I know) make an impact either way. Quotations should help with some special characters too, though this probably isn't a problem here.